src/data-workspace/data-details-sidebar/comment.js

```diff
@@ -58,10 +61,16 @@ CommentOptionSelector.propTypes = {
     inputOnChange: PropTypes.func,
 }
 
-const CommentEditField = ({ comment, commentOptionSetId }) => {
-    const { input, meta } = useField('comment', {
+const CommentEditField = ({
+    comment,
+    commentOptionSetId,
+    onBlur,
+    unsavedComment,
+    commentId,
+}) => {
+    const { input, meta } = useField(`comment_${commentId}`, {
```

Comment:
Think it could be useful to have a helper-function to "generate" these, as they are hard-coded in two different components.